src/less/flex.less

```diff
@@ -60,7 +60,7 @@
  * @param {number} basis - See `.flex-basis`. Defaults to auto.
  * @initial {0 1 auto}
  */
-.flex(@grow, @shrink: 1, @basis: auto) {
+.flex(@grow: 0, @shrink: 1, @basis: auto) {
```

Comment:
Ah ok. I probably just didn't understand what this meant. I'll revert it back to what it was before.